cartographer_ros/cartographer_ros/map_builder_bridge.cc

```diff
@@ -175,6 +175,7 @@ visualization_msgs::MarkerArray MapBuilderBridge::GetTrajectoryNodeList() {
        ++trajectory_id) {
     const auto& single_trajectory_nodes = all_trajectory_nodes[trajectory_id];
     visualization_msgs::Marker marker;
+    marker.ns = "Trajectory " + std::to_string(trajectory_id);
     marker.id = marker_id++;
```

Comment:
Marker id is not equal to trajectory id because we split the trajectory into multiple markers when there are more than 16384 trajectory nodes.